public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -124,11 +126,21 @@ const getStyles = (theme: GrafanaTheme2) => ({
     & + & {
       margin: ${theme.spacing(2, 0)};
     }
+
     position: relative;
   `,
+  footerSectionStyles: css`
+    padding: ${theme.spacing(1)};
+    background-color: ${theme.colors.background.primary};
+    position: sticky;
+    bottom: -24px; /* offset the padding on modal */
```

Comment:
Couldn't we use multiples of `theme.spacing(n)`?

Reply:
Good call, fixed and pushed.